test/e2e/storage/in_tree_volumes.go

```diff
@@ -53,7 +53,7 @@ var testDrivers = []func() storageframework.TestDriver{
 
 // This executes testSuites for in-tree volumes.
 var _ = utils.SIGDescribe("In-tree Volumes", func() {
-	if enableGcePD := os.Getenv("ENABLE_STORAGE_GCE_PD_DRIVER"); enableGcePD == "yes" {
+	if TestContext.EnableGcePdDriver {
```

Comment:
Maybe for now, support both env var and flag to give people time to fix their CI config.

Reply:
Sure (although I'd be willing to bet that the only configs changed were the kubetest2 ones I did where this wasn't actually effective...)

Reply:
I noticed openshift updated CI to use the envs.